res/TensorFlowLiteRecipes/BroadcastTo_001/test.recipe

```diff
@@ -0,0 +1,27 @@
+# We have to use the latest version of 3 for BroadcastTo Op.
+# https://github.com/tensorflow/tensorflow/blob/ce2d218fc88c20b1d81abb017eabd436779865e4/tensorflow/lite/kernels/register.cc#L306-L311
```

Comment:
@seanshpark 

> OK, thanks. Please add a comment that we have to use version 2 for this Op.

To mark this, I wrote annotations on the recipe file. 
Should I mark it on other parts? :)

Reply:
> Should I mark it on other parts? :)

Sorry but I don't understand your question.

> We have to use the latest version of 3 for BroadcastTo Op.

Why is it so? why not 2? URL you've added doesn't seem to explain this.

My comment about "we have to use version 2" is that this Op is the first Op that requires version > 1. I wanted some explanation what you understand about this.

Reply:
I've found related PR on tensorflow. (https://github.com/tensorflow/tensorflow/pull/46224/files)

```cxx
// The version one of broadcast to op won't be not supported since the
    // version one was rollbacked and the builtin op code number has been
    // changed because of builtin op code shortage problem.
    // Quantized broadcast_to is version 3
    case BuiltinOperator_BROADCAST_TO:
      if (op_sig.inputs.at(0).type == kTfLiteInt8 ||
          op_sig.inputs.at(0).type == kTfLiteInt16) {
        return 3;
      }
      return 2;
```

We are using tflite kernels for float values. So, the version should be 2.